apps/drupal/particle.theme

```diff
@@ -87,3 +89,39 @@ function particle_preprocess_page(&$variables) {
   }
 
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function particle_theme_suggestions_form_alter(array &$suggestions, array $variables) {
+  $id = $variables['element']['#id'];
+  if ($id == 'views-exposed-form-search-page' || $id == 'search-block-form') {
+    $suggestions[] = 'form__particle_search';
```

Comment:
Comment here that the file is drupal-only, located at `yaddayadda/themes/forms/form--particle-search.twig`